pliers/scikit.py

```diff
@@ -0,0 +1,41 @@
+from pliers.extractors import Extractor, merge_results
+
+try:
+    from sklearn.base import TransformerMixin, BaseEstimator
+except:
+    TransformerMixin, BaseEstimator = None
+
+
+class PliersTransformer(BaseEstimator, TransformerMixin):
+
+    ''' Simple wrapper for using pliers within a sklearn workflow.
+    Args:
+        transformer (Graph or Transformer): Pliers object to execute. Can
+            either be a Graph with several transformers chained or a single
+            transformer.
+    '''
+
+    def __init__(self, transformer):
```

Comment:
We could also probably allow it to take a string as the `transformer` argument, in which case it initializes the corresponding `Transformer` with all default arguments. Could save user a line of code in many cases.